### PR TITLE
Fix: Use appropriate sections for Travis configuration

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] uses appropriate sections to run things during the Travis build

💁‍♂️ Also adds a bit of vertical whitespace for enhanced legibility! For reference, see https://docs.travis-ci.com/user/job-lifecycle/#the-job-lifecycle.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,15 @@
 language: php
+
 php:
   - 7.1
   - 7.2
-before_script:
+
+before_install:
   - composer self-update
```
Comment (dereuromark) [inline .travis.yml]: This is not needed, travis has this included.

**Comment by ondrejmirtes** [comment]
This has a conflict now :(

**Comment by dereuromark** [comment]
This Pr is not needed. See current travis output. already runs as first thing.

**Comment by localheinz** [comment]
@dereuromark 

Not sure what you mean, can you explain?

File: .travis.yml (hunk 1 above)
Comment (localheinz) [reply .travis.yml]: See https://github.com/phpstan/phpstan-strict-rules/pull/72.

**Comment by dereuromark** [comment]
Travis already runs it itself.

**Comment by dereuromark** [comment]
Oh i am sry, the diff didnt show everything on mobile. Never mind.

**Comment by ondrejmirtes** [comment]
Closing as the path forward is rather in GitHub Actions :)

**Pull request closed without merging**